### PR TITLE
Add org.gnome.Loupe

### PR DESCRIPTION
**Pull request by sophie-h**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by sophie-h** [comment]
bot, build org.gnome.Loupe

**Comment by sophie-h** [comment]
bot, build org.gnome.Loupe

**Comment by sophie-h** [comment]
bot, build org.gnome.Loupe

**Comment by sophie-h** [comment]
Please add @brainblasted as admin for the Loupe Flathub repo as well after acceptance

File: org.gnome.Loupe.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.la'
+  - '*.a'
+add-extensions:
+  # Exile HEIC (x265) codec into extension
+  org.gnome.Loupe.Devel.HEIC:
```
Comment (BrainBlasted) [inline org.gnome.Loupe.yml:35]:
```suggestion
  org.gnome.Loupe.HEIC:
```

Or maybe we want `org.gnome.Loupe.Decoder.HEIC`?

Comment (sophie-h) [reply org.gnome.Loupe.yml]: Thanks!

File: org.gnome.Loupe.yml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  - org.freedesktop.Sdk.Extension.llvm15
+command: loupe
+finish-args:
+  # Graphics
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  # Browse through photos
+  - --filesystem=host
```
Comment (hfiguiere) [inline org.gnome.Loupe.yml:16]: this is very broad as a permission.

File: org.gnome.Loupe.yml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.la'
+  - '*.a'
+add-extensions:
+  # Exile HEIC (x265) codec into extension
+  org.gnome.Loupe.HEIC:
```
Comment (hfiguiere) [inline org.gnome.Loupe.yml:35]: what's the point of this?

File: org.gnome.Loupe.yml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+      - type: git
+        url: https://github.com/strukturag/libde265.git
+        commit: e255f0571a46935955deef15808e52b18043788c
+        tag: v1.0.11
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - name: libheif
+    buildsystem: cmake
```
Comment (hfiguiere) [inline org.gnome.Loupe.yml:62]:
```suggestion
    buildsystem: cmake-ninja
```

File: org.gnome.Loupe.yml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+    # Relative to directory above
+    add-ld-path: lib
+    # Remove extension data from main flatpak package
+    bundle: true
+    # Remove this extension when Loupe is deinstalled
+    autodelete: true
+modules:
+  # HEIC (x265) decoder
+  - name: libde265
+    buildsystem: cmake
```
Comment (hfiguiere) [inline org.gnome.Loupe.yml:47]:
```suggestion
    buildsystem: cmake-ninja
```

File: org.gnome.Loupe.yml (hunk 3 above)
Comment (sophie-h) [reply org.gnome.Loupe.yml:35]: <https://en.wikipedia.org/wiki/High_Efficiency_Video_Coding#Patent_license_terms>

Comment (hfiguiere) [reply org.gnome.Loupe.yml:35]: and?

File: org.gnome.Loupe.yml (hunk 2 above)
Comment (sophie-h) [reply org.gnome.Loupe.yml:16]: There hasn't been any progress on the relevant portal for years https://github.com/flatpak/xdg-desktop-portal/issues/463

People don't have images only in their Pictures folder. They also have graphics they are working on etc and want to flip through. Also, they want to open them from the file browser or terminal.

File: org.gnome.Loupe.yml (hunk 3 above)
Comment (sophie-h) [reply org.gnome.Loupe.yml:35]: We were kindly asked to allow everyone to opt out of codecs that might require licensing when used.

**Comment by sophie-h** [comment]
Works for me with the build from here. I suggest tracking bug reports in Loupe's issue tracker.

[Screencast from 2023-04-05 15-26-22.webm](https://user-images.githubusercontent.com/3466497/230094590-96190e0a-9dc9-45ca-bf80-6edf311fea03.webm)

**Pull request closed without merging**